data/human/human missions.txt

```diff
@@ -11281,3 +11281,164 @@ mission "Patch Stone Waiting"
+	choice
+		`	(Remain silent.)`
+		`	"Who are these weapons going to?"`
+			goto pact
+		`	"What's this 'developing situation?'"`
+			goto situation
+
+	`	After a moment of silence, one of the other captains steps forward and asks sneeringly, "How do we know these weapons aren't going to be used by illegal actors, my good quartermaster? Weapons trafficking can be a serious crime in the Republic."`
+	`	"You have no reason for suspicion," the quartermaster responds. "The weapons are being sent to our partners in the Southern Mutual Defense Pact, who have a much greater need for these munitions and outfits than we do, given the rates of piracy they're suffering through at the moment. We wouldn't need to do anything if it weren't for the Navy and their masters in Parliament sitting on their cushioned chairs every day and refusing to do anything about the situation while southron lives and ships are lost."`
+	`	The air in the office seems to become more frigid as the captain responds, "I asked you whether these weapons wouldn't fall into the hands of privateers and inciters like the Pact, and instead all you do is insult the Navy and tell us we should go help the Pact because Parliament made you do so. You begging southerners should be grateful for the Navy's protection!" He turns around and pushes through the still crowd in the room. A few others turn and follow him, adding, "This is all going to some rebellion, and you know it! These Southern Pact goons are gun-runners!"`
+	`	Going by their accents and faces, those captains came from the Paradise Worlds, and they didn't take the quartermaster's comment about the Navy or the Republic so kindly.`
+		goto decision
+
+	label situation
+	`	"The pirates have been becoming more and more aggressive in the South. They've started doing more than just interdict merchants; they've been raiding planets for ships and tribute on an organized scale once again, and the Pact has requested more weaponry to deal with them. Let me tell you though - we wouldn't have this problem if the Navy would stop hiding in the Paradise Worlds and start patrolling these regions again."`
+		goto raucous
+
+	label pact
+	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us some way in another and so we are sending our surplus to them in support of their situation."`
```

Comment:
Probably this would be best:

```suggestion
	`	"Many of the ships in our militia have affiliations with the Southern Mutual Defense Pact, which is a collection of planetary militia organizations. They're centered in the Rim and the South, so we are not directly tied to them, but recently we have begun to cooperate with them as the Navy has been cowering out in the Paradise Worlds. Whatever is going on down south that affects the Pact will also affect us in some way or another, so we are sending our surplus to them in support of their situation."`
```